res/graphics/quest icons/halal.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
+    <path d="m128 64c0 35.346-28.654 64-64 64s-64-28.654-64-64 28.654-64 64-64 64 28.654 64 64" fill="#e9a76f" stroke-width=".2"/>
+    <path d="m64.015 14.08c-28.702 0-51.971 23.269-51.971 51.971s23.269 51.971 51.971 51.971 51.971-23.269 51.971-51.971-23.269-51.971-51.971-51.971z" fill-opacity=".2"/>
+    <g transform="matrix(2.8872,0,0,-2.8872,12.074,113.99)">
+        <path d="m36 18c0-9.941-8.059-18-18-18s-18 8.059-18 18 8.059 18 18 18 18-8.059 18-18" fill="#fff"/>
+        <circle transform="scale(1,-1)" cx="17.985" cy="-18.007" r="13.854" fill="#444" fill-rule="evenodd" stroke-width=".039002"/>
+    </g>
+    <g fill="#fff" stroke-width="11.2" aria-label="حلال" font-size="171.34">
+        <path d="M100.63 69.7H75.06v-3.85H93.1q-2.26-1.75-3.8-2.73-1.93-1.21-3.52-1.77-1.8-.62-3.63-.62-1.13 0-2.28.16.64-1.64 1.68-2.63 1.5-1.44 3.8-1.44 2.26 0 5.34 2.21 1.53 1.1 4.98 4.2 2.68 2.42 3.63 2.62h1.33z"/>
+        <path d="M67.45 58.12q0 .44-.05.75-.04.31-.22.78-1.24-2.77-3.67-5.71-2.7-3.26-4.25-3.26-.33 0-.67.58-.33.58-.77.58-.6 0-1.88-.78-1.62-.97-2.6-2.26-1.3-1.72-1.3-3.67 0-1.4.9-1.4 1.6 0 4.99 2.02 3.78 2.23 6.35 5.22 3.17 3.65 3.17 7.15zM76.6 69.7h-1.35q-1.86 0-2.99-2.02-1.35-2.43-1.35-7.37 0-.35-.09-.95-.06-.6-.06-.75-2.13 5.9-3.79 8.1-1.97 2.57-5.55 3.41-3.06.7-10.4.7v-.61q5.95-1.8 8.87-3.1 4.71-2.08 7.15-4.8 2.94-3.28 2.94-7.77 0-2.17-.28-4.05-.27-1.89-.76-3.86l.73-1.86q.45-1.08.82-1.83.58 3.18 1.09 6.88.35 2.55.68 5.47.42 3.58.73 5.78.4 2.79 1.2 3.98.53.8 1.06.8h1.35zm-27.21-2.75q0 3.55-1.67 5.63-1.48 1.83-4.36 2.63-2.43.66-6.17.66-4.27 0-6.62-1.57Q28 72.58 28 69.15q0-1.9.56-3.97.46-1.75 1.41-3.83.69-1.5 1.86-3.63l.71.42q-.82 1.49-1.37 2.66-.73 1.57-1.09 2.81-.42 1.5-.42 2.92 0 2.73 1.97 4.12 2 1.4 5.51 1.4 3.75 0 6.93-1.29 3.5-1.42 3.5-3.39 0-2.14-.69-7.28-.53-3.94-1.3-8.8l-.73-4.3-1.02-.53q0-2.97 1.22-5.4h.37q.1.84.31 1.26.22.4.73.6.69.28 1.5.68l1.05.51q0 1.55-.11 2.26-.11.7-.44 1.99l-1.27-.4q.05.49.65 4.45.68 4.63 1.01 7.5.54 4.6.54 7.04z"/>
+    </g>
+</svg>
```

Comment:
Good!

Though, the black and white "K"-letter was inspired by some label I saw on the internet. If any color is usually associated with Halal (e.g. green in the SVG that was posted earlier), it could be green too. When I wrote that it should be "like" the kashrut-icon

Reply:
As far as I know - green is general color associated with islam, so green would be likely fitting.

(and kosher logos were quite often black, so I went with black style quite similar to a typical one)